health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/Inbox.js

```
@@ -0,0 +1,63 @@
+    InboxConfig?.InboxConfig?.map((configItem, index) => ({
+      key: index,
+      label: configItem.label,
+      active: index === 0,
+    }))
+  );
+
+  useEffect(() => {
+    let savedIndex = Digit.SessionStorage.get("HCM_SELECTED_TAB_INDEX");
+
+    // If not set, default to 0 and save it in sessionStorage
+    if (savedIndex === null || isNaN(parseInt(savedIndex))) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)